core/pom.xml

```diff
@@ -398,6 +398,41 @@
       <artifactId>py4j</artifactId>
       <version>0.8.2.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>${avro.version}</version>
+      <scope>${hadoop.deps.scope}</scope>
```

Comment:
`avro` and `avro-mapred` are already in the `dependencyMangement` of the parent POM, so nothing more than `groupId`, `artifactId` and perhaps `scope` should be included here.  In particular, `version` should not generally be specified when it is already in the `dependencyManagement`.  This is a frequent error in the Spark POMs.